### ISSUE TITLE
WebJar Request: davidjbradshaw/iframe-resizer

### ISSUE THREAD

**Issue by rgeoff** [webjars/webjars#1239]
```
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
    <modelVersion>4.0.0</modelVersion>

    <parent>
        <groupId>org.sonatype.oss</groupId>
        <artifactId>oss-parent</artifactId>
        <version>7</version>
    </parent>
    
    <packaging>jar</packaging>
    <groupId>org.webjars</groupId>
    <artifactId>davidjbradshaw/iframe-resizer</artifactId>
    <version>3.4.2-SNAPSHOT</version>
    <name>iframe-resizer</name>
    <description>WebJar for iframe-resizer</description>
    <url>http://webjars.org</url>

    <properties>
        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
        <upstream.version>3.4.2</upstream.version>
        <upstream.url>https://github.com/davidjbradshaw/iframe-resizer/archive/v${upstream.version}.zip</upstream.url>
        <destDir>${project.build.outputDirectory}/META-INF/resources/webjars/${project.artifactId}/${project.version}</destDir>
        <requirejs>
            {
                "paths": {
                    "davidjbradshaw/iframe-resizer": "davidjbradshaw/iframe-resizer"
                },
                "shim": {
                    "davidjbradshaw/iframe-resizer": ["DEP_ID"]
                }
            }
        </requirejs>
    </properties>

    <developers>
        <developer>
            <id>rgeoff</id>
            <name></name>
            <email></email>
        </developer>
    </developers>

    <licenses>
        <license>
            <name>MIT</name>
            <url>https://github.com/davidjbradshaw/iframe-resizer/blob/master/LICENSE</url>
            <distribution>repo</distribution>
        </license>
    </licenses>

    <scm>
        <url>http://github.com/webjars/davidjbradshaw/iframe-resizer</url>
        <connection>scm:git:https://github.com/webjars/davidjbradshaw/iframe-resizer.git</connection>
        <developerConnection>scm:git:https://github.com/webjars/davidjbradshaw/iframe-resizer.git</developerConnection>
        <tag>HEAD</tag>
    </scm>
    
    <dependencies>
        <dependency>
            <groupId>org.webjars</groupId>
            <artifactId>ANOTHER_WEBJAR_ARTIFACT_ID</artifactId>
            <version>ANOTHER_WEBJAR_ARTIFACT_VERSION</version>
        </dependency>
    </dependencies>
    
    <build>
        <plugins>
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-antrun-plugin</artifactId>
                <version>1.8</version>
                <executions>
                    <execution>
                        <phase>process-resources</phase>
                        <goals><goal>run</goal></goals>
                        <configuration>
                            <target>
                                <echo message="download archive"/>
                                <get src="${upstream.url}" dest="${project.build.directory}/${project.artifactId}.zip"/>
                                <echo message="unzip archive"/>
                                <unzip src="${project.build.directory}/${project.artifactId}.zip" dest="${project.build.directory}"/>
                                <echo message="moving resources"/>
                                <move todir="${destDir}">
                                    <fileset dir="${project.build.directory}/${project.artifactId}-${upstream.version}"/>
                                </move>
                            </target>
                        </configuration>
                    </execution>
                </executions>
            </plugin>

            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-release-plugin</artifactId>
                <version>2.5.3</version>
            </plugin>

            <plugin>
                <groupId>org.sonatype.plugins</groupId>
                <artifactId>nexus-staging-maven-plugin</artifactId>
                <version>1.6.6</version>
                <extensions>true</extensions>
                <configuration>
                    <serverId>sonatype-nexus-staging</serverId>
                    <nexusUrl>https://oss.sonatype.org/</nexusUrl>
                    <autoReleaseAfterClose>true</autoReleaseAfterClose>
                </configuration>
            </plugin>
        </plugins>
    </build>

</project>

```

**Comment by jamesward** [comment]
This has been deployed as a Bower WebJar:
```
GroupID = org.webjars.bower
ArtifactID = iframe-resizer
Version = 3.4.2
```

**Comment by rgeoff** [comment]
Hi James,

We need a version deployed as a classic webjar.

Thanks

**Comment by jamesward** [comment]
Ok.  But curious as to what doesn't work about the Bower WebJar for this?

**Comment by rgeoff** [comment]
Yeah, I agree. Just more of a request for a non-Bower project. I appreciate the help, thanks.

**Comment by jamesward** [comment]
This already exists: https://github.com/webjars/iframe-resizer

But I've released a new version: `3.4.2`

In the future please file version bumps for existing WebJars on the actual project.

**Comment by rgeoff** [comment]
Okay

Thanks James
